config/config.go

```diff
@@ -18,24 +18,21 @@ import (
 )
 
 const (
-	//Contains the Authentication address that will be used to validate all requests to update Daemon configuration remotely through a user interface
-	AuthenticationAddress     = "authentication_address"
-	AutoSSLDomainKey          = "auto_ssl_domain"
-	AutoSSLCacheDirKey        = "auto_ssl_cache_dir"
-	BlockchainEnabledKey      = "blockchain_enabled"
-	BlockChainNetworkSelected = "blockchain_network_selected"
-	BurstSize                 = "burst_size"
-	ConfigPathKey             = "config_path"
-    CurationAddressForValidation   = "curation_address_for_validation"
+	AllowedUserFlag                = "allowed_user_flag"
```

Comment:
Do we need to have the additional flag, cant we manage with the allowedusernames field. Tough for admin to update two configurations...